ReactWindows/ReactNative.Shared/Bridge/ReactContext.cs

```diff
@@ -326,7 +326,7 @@ public void HandleException(Exception exception)
         /// <remarks>
         /// This method should be called exactly once.
         /// </remarks>
-        internal void InitializeWithInstance(IReactInstance instance)
+        public void InitializeWithInstance(IReactInstance instance)
```

Comment:
We're using this in the tests and without it being public VS complains. Not sure what black magic occurs in the UWP tests that allows it to get around this requirement but on that side it doesn't seem to care.